modules/oe_content_event/config/install/field.field.node.oe_event.field_registration_status.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_registration_status
+    - node.type.oe_event
+  module:
+    - options
+id: node.oe_event.field_registration_status
+field_name: field_registration_status
+entity_type: node
+bundle: oe_event
+label: 'Registration status'
+description: ''
+required: false
```

Comment:
should be required

Reply:
Done through constraint. By documentation this field is required only if any fields within Registration fields group filled in.